From 1abbd6717719072418fa08274e586ba9e8d1dff9 Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Fri, 21 Feb 2020 23:47:47 +0200
Subject: [PATCH 7/7] try to add FM radio

	modified:   target/board/BoardConfigGsiCommon.mk
	modified:   target/product/mainline.mk

Change-Id: I3616d333d3614ffe3add1d19330570d30e58dd17
---
 target/board/BoardConfigGsiCommon.mk | 4 ++++
 target/product/mainline.mk           | 2 ++
 2 files changed, 6 insertions(+)

diff --git a/target/board/BoardConfigGsiCommon.mk b/target/board/BoardConfigGsiCommon.mk
index 702ef3c..ba4d013 100644
--- a/target/board/BoardConfigGsiCommon.mk
+++ b/target/board/BoardConfigGsiCommon.mk
@@ -16,6 +16,10 @@ BOARD_USES_SYSTEM_OTHER_ODEX :=
 # for some devices
 TARGET_USERIMAGES_USE_F2FS := true
 
+# FM
+BOARD_HAVE_QCOM_FM := true
+TARGET_QCOM_NO_FM_FIRMWARE := true
+
 # Enable dynamic system image size and reserved 64MB in it.
 BOARD_SYSTEMIMAGE_PARTITION_RESERVED_SIZE := 67108864
 
diff --git a/target/product/mainline.mk b/target/product/mainline.mk
index 20039b9..d018591 100644
--- a/target/product/mainline.mk
+++ b/target/product/mainline.mk
@@ -40,3 +40,5 @@ endif
 PRODUCT_PACKAGES += \
     PhotoTable \
     WallpaperPicker \
+    FMRadio \
+    libfmjni \
-- 
2.17.1

